flatdata-py/README.md

```diff
@@ -0,0 +1,14 @@
+# flatdata-py
+
+Python 3 implementation
+
+
+## Basic usage
+
+Once you have [created a flatdata schema file](../README.md#creating-a-schema), you can generate a Python module:
+
+```sh
+./generator/app.py --gen python --schema locations.flatdata --output-file locations.py
+```
+
+TODO: Add basic python usages
```

Comment:
Just a note: There is also inspect flatdata file in tools. I would actually take a look at this file and consider to provide instructions instead how to use flatdata from jupyter.

Reply:
Sounds like a good idea. In general, we need to check why `flatdata-py` depends on the internal modules of the `generator` - with this proper split of directories, you either need to run those scripts from the root, or manipulate `PYTHONPATH`, which I don't like very much (it's never very obvious).

Reply:
flatdata-py is 90% generator and just a little bit on top if I am not mistaken (it basically uses the AST the generator has internally)

Reply:
then do you think it makes sense to merge back the generator inside of flatdata-py and have a symlink at the root, or a launcher called `generator` or `generator.py`?